### PR TITLE
Fix apigee addons config conditional expression

### PR DESCRIPTION
**Pull request by eddern**
I assume the intended behavior of the `addons_config` in the Apigee module is to enable the addons that are set to `true`?

As it is now, I need to pass this in order to enable connectors and integration:
```
addons_config = {
    connectors_platform = false
    integration         = false
    api_security = true
    monetization = true
    advanced_api_ops = true
 }
```

---
**Checklist**

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [ ] Made sure all relevant tests pass

### CODE REVIEW

File: modules/apigee/main.tf [hunk 1]
```diff
@@ -137,31 +137,31 @@ resource "google_apigee_addons_config" "addons_config" {
   org      = local.org_name
   addons_config {
     dynamic "advanced_api_ops_config" {
-      for_each = var.addons_config.advanced_api_ops ? [] : [""]
+      for_each = var.addons_config.advanced_api_ops ? [""] : []
       content {
         enabled = true
```
Comment (juliocc) [inline modules/apigee/main.tf:142]: can we us the actual value here? 

The following should be enough:
```hcl
      advanced_api_ops_config {
        enabled = var.addons_config.advanced_api_ops
      }
```

And same for all the other options.

**Review by juliocc — CHANGES_REQUESTED** [review]

File: modules/apigee/main.tf (hunk 1 above)
Comment (eddern) [reply modules/apigee/main.tf]: Sure thing. 

Much cleaner like you suggested 👍

**Review by juliocc — APPROVED** [review]

**Comment by eddern** [comment]
@juliocc after upgrading my Apigee module to the newest version, terraform keeps wanting to update the values to false every time I plan (even though I have applied several times). Any idea why this is?

Screenshot 2023-10-23 at 09 20 06

**Comment by eddern** [comment]
The same applies with all the values explicitly defined in the config as well
Screenshot 2023-10-23 at 09 31 10

**Comment by juliocc** [comment]
Looks like my suggestion of using the value directly in the configuration block doesn't work. Apparently the provider is only expecting values to be true or nothing.

Do you mind reverting to the previous syntax.

**Comment by mwarm2** [comment]
See #1798

**Pull request merged**